### DISCUSSION TITLE
A custom delay between each block placement in the config

### DISCUSSION THREAD

**Discussion by JDLanyon** [maruohon/tweakeroo#358] [Ideas]
Not sure if this is already an option, apologies in advanced if it is.

Playing on public servers and servers with a bit of ping can prove a little difficult when using fast block placements.
I was wondering if it would be possible for a time variable in the config menu to add a delay between each block placement.
The issue I've had is either ghost blocks being quite prevalent or the server receives a fair bit more packets than what it likes.

**Comment by maruohon** [comment]
It's quite likely that this will get added during the rewrite of all the placement related code in the somewhat near future.
